src/rdb_threads.h

```diff
@@ -0,0 +1,48 @@
+#ifndef __RDB_THREADS_H__
+#define __RDB_THREADS_H__
+
+#include "server.h"
+#include "thread_common.h"
+
+/* Threshold for flushing a worker's buffer to the main RDB file (64 KB). */
+#define WORKER_BUFFER_DEFAULT_SIZE 64 * (1024)
+/* Maximum capacity for a worker's buffer. Keys causing this limit to be exceeded are streamed directly to RDB file (256 KB). */
```

Comment:
Maybe explain more about the big key scenario? Why do we need two thresholds, and how do they solve the big key case.